### PR TITLE
Making the yo symphony generator a wrapper for yo @finos/symphony

### PR DESCRIPTION
**Pull request by symphony-mariacristina**
### Description
Symphony generator has been moved under Finos scope, this means that now in order to call latest versions of the generator we have to type the following command "yo @finos/symphony".
Goal of this implementation is to release a new package 2.4.3 that is going to report to the user that the package as been moved and it's going to automatically use the new command.

Screenshot 2022-03-09 at 12 57 34

### Checklist
- [x] Referenced an issue in the PR title or description
- [x] Filled properly the description and dependencies, if any
- [x] Unit/Integration tests updated or added
- [x] Javadoc added or updated
- [x] Updated the documentation in [docs folder](../docs)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,24 +16,8 @@
...
-  },
-  "devDependencies": {
-    "decamelize": "^5.0.0",
-    "fs-extra": "^10.0.0",
-    "jest": "^27.2.0",
-    "jest-junit": "^12.0.0",
-    "jest-mock-axios": "^4.4.0",
-    "yeoman-assert": "^3.1.1",
-    "yeoman-test": "^6.2.0"
+    "@finos/generator-symphony": "latest"
```
Comment (symphony-mariacristina) [inline package.json:20]: I put latest here but we could also use 2.5.x

**Comment by symphony-mariacristina** [comment]
should I commit the yarn.lock file as well?

**Review by symphony-youri — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (symphony-thibault) [reply package.json:20]: For me it's ok to use `latest`

**Review by symphony-thibault — APPROVED** [review]

**Comment by symphony-youri** [comment]
Maybe we can update the README to explain that we use a dependency and execute the new generator instead?
And is it the branch we want to merge to? Maybe we should give a better name like 2.4-rc ?

**Review by symphony-elias — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
 [![FINOS - Incubating](https://cdn.jsdelivr.net/gh/finos/contrib-toolbox@master/images/badge-incubating.svg)](https://finosfoundation.atlassian.net/wiki/display/FINOS/Incubating)
 [![npm version](https://badge.fury.io/js/generator-symphony.svg)](https://badge.fury.io/js/generator-symphony)
 [![CircleCI](https://circleci.com/gh/SymphonyPlatformSolutions/generator-symphony.svg?style=shield)](https://circleci.com/gh/SymphonyPlatformSolutions/generator-symphony)
+
+## ⚠️ Important notice ⚠️
```
Comment (symphony-youri) [inline README.md:5]: The README should be https://www.npmjs.com/package/generator-symphony today
But I guess it has not been committed and only changed locally...
Could take that one instead and just add this notice paragraph?

Comment (symphony-mariacristina) [reply README.md:3]: Done https://github.com/finos/generator-symphony/pull/187/commits/35a8db10ae807eb1ae24d7c923effc4d52ea1b76

**Review by symphony-youri — APPROVED** [review]

**Pull request merged**